compiler/test/dotty/tools/vulpix/Status.scala

```diff
@@ -0,0 +1,7 @@
+package dotty.tools
+package vulpix
+
+sealed trait Status
+final case class Success(output: String) extends Status
+final case class Failure(output: String) extends Status
+final case object Timeout extends Status
```

Comment:
What's the difference between Timeout and Failure("Process timed out after ..."), do we really need a special case for Timeout ?